conf/application.conf

```diff
@@ -185,6 +185,8 @@ urls {
   locationCodesForCsePremises = "https://www.gov.uk/government/publications/location-codes-to-declare-goods-for-export-at-cse-premises-for-data-element-523-of-the-customs-declaration-service"
   goodsLocationCodesForDataElement = "https://www.gov.uk/government/collections/goods-location-codes-for-data-element-523-of-the-customs-declaration-service"
   tariffCdsChiefSupplement = "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/956071/Tariff_Volume_3_Great_Britain_Supplement_CDS_and_CHIEF_version_5.3__1_.odt"
+  ducrToMucrHint1 = "https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-export-declaration-completion-guide/group-2-references-of-messages-document-certificates-and-authorisations#de-21-simplified-declaration-previous-documents-box-40-declaration-previous-document"
+  ducrToMucrHint2 = "https://www.gov.uk"  // To replace. It will be provided through another ticket
```

Comment:
IMHO these two fields should be named based on where they point to. The name `ducrToMucrHintX` is good, but maybe somewhere deeper in the code, for e.g. fields in `AppConfig`.